src/server/repository/public/user/getOne.ts

```diff
@@ -1,23 +1,43 @@
 import { Objects } from 'utils/objects'
 
+import { Areas } from 'meta/area'
 import { User, UserStatus } from 'meta/user'
 
 import { BaseProtocol, DB } from 'server/db'
 import { UserRoleAdapter } from 'server/repository/adapter'
-import { ProcessEnv } from 'server/utils'
 
 import { fields } from './fields'
 
 const selectFields = fields.map((f) => `u.${f}`).join(',')
 
+const userRolesSubquery = `
```

Comment:
The issue mentions
> line 65: get rid of usersRoleSubquery

But I think it is required if we want to exclude Atlantis roles in published cycles

Let me know